Packs/CortexAttackSurfaceManagement/pack_metadata.json

```diff
@@ -80,6 +80,10 @@
         "GCP-IAM": {
             "mandatory": false,
             "display_name": "GCP-IAM"
+        },
+        "MicrosoftGraphUser": {
+            "mandatory": false,
+            "display_name": "Microsoft Graph User"
```

Comment:
Was this required for validation?

Reply:
Yeah. It is throwing a dependency validation error if we don't add.